gui/create-item.lua

```diff
@@ -255,15 +259,38 @@ function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, ge
   item_type = "GLOB"
  elseif CORPSE_PIECES[layerName] or HAIR_PIECES[layerName] then -- check if hair
   item_type = "CORPSEPIECE"
+ elseif LIQUID_PIECES[layerName] then -- check if hair
+  item_type = "LIQUID_MISC" -- prolly need to do more work on this (it doesn't like spawning LIQUID items), but it works for now
```

Comment:
what exactly doesn't work, and what do you think could be done to make it better? it might be worthwhile to create an issue on GitHub with any follow-up plans you have

Reply:
I can get it